### PR TITLE
Update todo

### PR DESCRIPTION
**Pull request by royvanderzon**
Added 3 todo's.

### CODE REVIEW

File: todo.md [hunk 1]
```diff
@@ -61,6 +61,7 @@
 
 *   Prevent multiple versions and requests for jQuery. The page itself requests jQuery, but the plugins also include different versions.
 *   Get rid of js files that aren't used. JS takes up ~50% of the requests and bytes
+*   Use of webworkers
```
Comment (wooorm) [inline todo.md:64]: web workers or service workers?

Comment (wooorm) [reply todo.md:64]: ...or both?

File: todo.md [hunk 2]
```diff
@@ -155,6 +157,7 @@
 
 *   Build custom theme
 *   Reserve space for the full size image on the home page. When the page is loaded, the content won't jump around
+*   Wordpress minify assets https://wordpress.org/plugins/bwp-minify/
```
Comment (wooorm) [inline todo.md]: Could you use the markdown link format? E.g., `[WordPress minify assets](https://wordpress.org/plugins/bwp-minify/)`

**Review by wooorm — CHANGES_REQUESTED** [review]

File: todo.md (hunk 1 above)
Comment (royvanderzon) [reply todo.md:64]: Webworkers, because service workers where already in the list.

Comment (wooorm) [reply todo.md:64]: :+1:

**Review by wooorm — APPROVED** [review]

**Comment by wooorm** [comment]
@royvanderzon Awesome, thanks!

**Pull request merged**